lib/configSchema.js

```diff
@@ -14,6 +14,7 @@ const schema = {
       additionalProperties: false,
     },
     frameworkVersion: { type: 'string' },
+    useDotenv: { const: true },
```

Comment:
Let's put it in alphabetical order (I know this set was not ordered that way, I've just fixed it with: https://github.com/serverless/serverless/pull/8460)